File: web-client/src/types/api/runtime-target.ts
/**
 * Runtime Target 接口的原始响应类型。
 */
export interface RuntimeTargetResponse {
    id: number;
    runtime_type: string;
    runtime_instance_id: number;
    runtime_profile_id: number;
    target_key: string;
    display_name: string;
    role_name: string | null;
    cs_id: string;
    enabled: boolean;
    instance_name: string;
}

File: scheduler-server/src/schemas/runtime_target.py
"""Runtime Target 对外读取结构。"""

from pydantic import BaseModel


class RuntimeTargetRead(BaseModel):
    id: int
    runtime_type: str
    runtime_instance_id: int
    runtime_profile_id: int
    target_key: str
    display_name: str
    role_name: str | None
    cs_id: str
    enabled: bool
    instance_name: str
